src/dashboard/src/requirements/base.txt

```diff
@@ -9,9 +10,11 @@ django-extensions==1.1.1
 django-autoslug==1.7.1
 django-annoying==0.7.7
 elasticsearch>=1.0.0,<2.0.0
-git+https://github.com/artefactual/archivematica-fpr-admin.git@v1.3.0-beta.1#egg=archivematica-fpr-admin
+git+https://github.com/artefactual/archivematica-fpr-admin.git@389453ce6ed2046d69c4b7fd6723628aca8c5abf#egg=archivematica-fpr-admin
```

Comment:
It's still preferably to use 389453ce6ed2046d69c4b7fd6723628aca8c5abf, I just forgot to commit it! Fixed now.

Here is the related pull request: https://github.com/artefactual/archivematica-fpr-admin/pull/37.